holidays/countries/brunei.py

```diff
@@ -377,52 +444,86 @@ class BruneiIslamicHolidays(_CustomIslamicHolidays):
         2009: (SEP, 7),
         2010: (AUG, 27),
         2011: (AUG, 17),
-        2012: (AUG, 5),
-        2013: (JUL, 25),
+        2012: (AUG, 6),
+        2013: (JUL, 26),
         2014: (JUL, 15),
         2015: (JUL, 4),
-        2016: (JUN, 23),
+        2016: (JUN, 22),
         2017: (JUN, 12),
         2018: (JUN, 2),
-        2019: (MAY, 23),
+        2019: (MAY, 22),
         2020: (MAY, 10),
         2021: (APR, 29),
         2022: (APR, 19),
         2023: (APR, 8),
         2024: (MAR, 28),
+        2025: (MAR, 18),
     }
 
     RAMADAN_BEGINNING_DATES = {
-        2000: (NOV, 28),
+        1998: (DEC, 20),
+        1999: (DEC, 9),
+        2000: (NOV, 27),
         2001: (NOV, 17),
         2002: (NOV, 6),
         2003: (OCT, 27),
         2004: (OCT, 16),
         2005: (OCT, 5),
         2006: (SEP, 24),
         2007: (SEP, 13),
-        2008: (SEP, 2),
+        2008: (SEP, 1),
         2009: (AUG, 22),
         2010: (AUG, 11),
         2011: (AUG, 1),
-        2012: (JUL, 20),
-        2013: (JUL, 9),
+        2012: (JUL, 21),
+        2013: (JUL, 10),
         2014: (JUN, 29),
         2015: (JUN, 18),
-        2016: (JUN, 7),
+        2016: (JUN, 6),
         2017: (MAY, 27),
-        2018: (MAY, 16),
+        2018: (MAY, 17),
         2019: (MAY, 6),
-        2020: (APR, 25),
+        2020: (APR, 24),
         2021: (APR, 13),
         2022: (APR, 3),
         2023: (MAR, 23),
         2024: (MAR, 12),
+        2025: (MAR, 2),
     }
 
 
 class BruneiStaticHolidays:
     special_public_holidays = {
-        # Sultan Hassanal Bolkiah's Golden Jubilee celebration
+        1998: (
+            AUG,
+            10,
+            # Proclamation Ceremony of Crown Prince Al-Muhtadee Billah of Brunei.
+            tr(
+                "Istiadat Pemasyhuran Yang Teramat Mulia Paduka Seri Duli Pengiran Muda Haji "
+                "Al-Muhtadee Billah Menjadi Duli Yang Teramat Mulia Paduka Seri Pengiran Muda "
+                "Mahkota"
```

Comment:
I found where it came from, but maybe somehow it can be shortened (this and especially next name)? 👀

Reply:
I'm thinking of simplifying both to "`Istiadat Pengisytiharan Duli Pengiran Muda Mahkota Al-Muhtadee Billah`" (Proclamation Ceremony of Crown Prince Al-Muhtadee Billah of Brunei) and "`Istiadat Perkahwinan Diraja Brunei 2004`" (Brunei Royal Wedding Ceremony 2004) for the latter in that case since including both the bride and groom name would be too long.